app/component-library/components-temp/CustomSpendCap/CustomSpendCap.tsx

```diff
@@ -75,8 +76,8 @@ const CustomSpendCap = ({
   };
 
   useEffect(() => {
-    if (maxSelected) setValue(accountBalance);
-  }, [maxSelected, accountBalance]);
+    if (maxSelected) setValue(unroundedAccountBalance || accountBalance);
+  }, [maxSelected, accountBalance, unroundedAccountBalance]);
```

Comment:
```suggestion
    if (maxSelected) setValue(unroundedAccountBalance);
  }, [maxSelected, unroundedAccountBalance]);
```

I think we may no longer need accountBalance. Else I think since unroundedAccountBalance may be an empty string, || accountBalance may never be reached